app/services/uri_service.rb

```diff
@@ -12,9 +12,10 @@ def self.params(original_uri)
   # @return [URI, nil]
```

Comment:
oh I just realized that the return value is a lie, with the `#to_s` on there we often return a string
```suggestion
  # @return [String, URI, nil]
```

Reply:
Hm, should we normalize this a bit? Could call `original_uri.to_s` in the new path I added

Reply:
yeah that would work!

Reply:
Updated to consistently return string in 3a09cdda57.